packages/lesswrong/components/comments/CommentsViews.tsx

```diff
@@ -20,41 +19,25 @@ export const viewNames: Partial<Record<CommentsViewName,string>> = {
   'postLWComments': 'top scoring (include LW)',
 }
 
-const styles = (theme: ThemeType): JssStyles => ({
-  root: {
-    display: 'inline'
-  },
-  link: {
-    color: theme.palette.lwTertiary.main,
-  }
-})
-
 const CommentsViews = ({post, classes}: {
   post?: PostsDetails,
   classes: ClassesType,
 }) => {
-  const [anchorEl,setAnchorEl] = useState<any>(null);
   const currentUser = useCurrentUser();
   const { history } = useNavigation();
   const location = useLocation();
   const { query } = location;
 
-  const handleClick = (event: React.MouseEvent) => {
-    setAnchorEl(event.currentTarget);
-  };
+  const {InlineSelect} = Components
 
-  const handleViewClick = (view: string) => {
+  const handleViewClick = (opt: Option) => {
```

Comment:
We could type this as `opt: Option & {value: CommentsViewName}` to avoid using `as` below